### DISCUSSION TITLE
Can someone make this easier for a layman to install?

### DISCUSSION THREAD

**Discussion by eru7771989** [manu12121999/RemoveBG-GIMP#14] [General]
I have a vague idea of what an API is, and the manual says I need to hardcode it into my code lol.
I just need to use this for work on gimp, to remove backgrounds. Is the API key paid for and if so , how much does it cost and how do I get one.
I am not a developer, so it would be nice to have instructions past the plugin installation phase.
I have installed the gimp plugin, and now it's asking me for an API key.

**Comment by manu12121999** [comment]
You don't need to write the API key it in the code. I just mentioned that you can do this for more convenience (so you don't have to put the key into the Popup inside gimp every time you use the plugin).
The API key is not paid, but limited uses (50 per month). You get it on [remove.bg/api], but you need to sign in on that website (e.g. with Google)

Hope this helps
